simple_stream.go

```diff
@@ -55,10 +58,18 @@ func (so *simpleStreamOpener) openStream() (streamer, error) {
 }
 
 func (so *simpleStreamOpener) streamURL(id string) string {
-	if so.batchLimit != 0 && so.flushTimeout != 30 {
-		return fmt.Sprintf("%s/subscriptions/%s/events?batch_limit=%d&batch_flush_timeout=%d", so.client.nakadiURL, id, so.batchLimit, so.flushTimeout)
+	queryParams := url.Values{}
+	if so.batchLimit > 0 {
+		queryParams.Add("batch_limit", fmt.Sprint(so.batchLimit))
```

Comment:
I would use `strconv.FormatUint` instead of `fmt.Sprint` here.

Reply:
Fixed